xsd/NX.xsd

```diff
@@ -37,7 +37,7 @@
 						<xsd:element name="points" minOccurs="0">
 							<xsd:complexType>
 								<xsd:sequence>
-									<xsd:element ref="Point" maxOccurs="unbounded"/>
+									<xsd:element ref="Point_" maxOccurs="unbounded"/>
```

Comment:
Are you sure about these changes? It feels like you are referencing to 'abstract' datatypes / 'containers'.

Reply:
I am an engineer (not true): It works... And it seems to me that they did it that way on other xxx_ elements.  What it does it references the "concrete" element that is used in the substitutionGroup. Otherwise it wouldn't work. But to be sure we would need a comment from Nick or Christophe.

Reply:
@nick-knowles @Aurige

Reply:
This is a bit different from the topic of the PR ;-) but technically should be Ok: I think that this is replacing a ref to anything inheriting from Point to anything being of in the Point_ substitution group. I didn't check the possible semantic impact (yet).